frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -7,7 +7,10 @@ import { useTranslation } from 'react-i18next';
 import { RepoStatus, ImportStrategy, getGitService, GitProvider } from '@console/git-service';
 import { DetectedBuildType } from '@console/git-service/src/utils/build-tool-type-detector';
 import { detectImportStrategies } from '@console/git-service/src/utils/import-strategy-detector';
+import { getActiveNamespace } from '@console/internal/actions/ui';
 import { BuildStrategyType } from '@console/internal/components/build';
+import { useAccessReview } from '@console/internal/components/utils';
+import { FLAG_KNATIVE_SERVING_SERVICE, ServiceModel } from '@console/knative-plugin/src';
```

Comment:
ideally would be nice to avoid import from knative package to dev-console

```suggestion
import { FLAG_KNATIVE_SERVING_SERVICE, ServiceModel as ksvcModel } from '@console/knative-plugin/src';
```